usecases/network_statistics_api.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from elasticsearch import exceptions
+
+from common.errors import ApiError
+from common.logging import get_logger
+from gateways.network_statistics_api_gateway import NetworkStatisticsApiGateway
+
+logger = get_logger()
+
+
+class NetworkStatisticsApi:
+    def __init__(
+        self,
+        network_statistics_api_gateway: Optional[NetworkStatisticsApiGateway] = None,
+    ) -> Optional[None]:
+        self.network_statistics_api_gateway = (
+            network_statistics_api_gateway or NetworkStatisticsApiGateway()
+        )
+
+    def get_basic_statistics(self) -> dict:
+        try:
+            return self.network_statistics_api_gateway.get_transaction_statistics()
+        except exceptions.ApiError as err:
+            logger.error(
+                "ExplorerService failed to fetch transaction statistics from ElasticSearch",
+                {"status": err.status_code, "msg": err.error},
+            )
+            raise ApiError("gateway_error")
+        except exceptions.TransportError:
+            logger.error("ElasticSearch transport error")
+            raise ApiError("internal_error")
```

Comment:
Wouldn't it be better to raise the original error message after writing to the logger here?

I feel we can be a bit lost debugging whenever these `TransportError`s occur, because their message has been suppressed.

Reply:
We can't raise the original error because it would expose internal information to the client. We can add the error to the error message in other PR, indeed.G